OpenRA.Game/FileSystem/ZipFile.cs

```diff
@@ -19,142 +19,227 @@
 
 namespace OpenRA.FileSystem
 {
-	public sealed class ZipFile : IReadWritePackage
+	public class ZipFileLoader : IPackageLoader
 	{
-		public IReadWritePackage Parent { get; private set; }
-		public string Name { get; private set; }
-		readonly Stream pkgStream;
-		readonly SZipFile pkg;
+		static readonly string[] Extensions = { ".zip", ".oramod", ".oramap" };
```

Comment:
I thought we dropped .oramod support, is it intentional that you keep support for that extension?